examples/overlay/messages/07_input_ngd.json

```diff
@@ -0,0 +1,63 @@
+{
+    "query_graph": {
+        "nodes": {
+            "type-2 diabetes": {
+                "id": "MONDO:0005148"
+            },
+            "drug": {
+                "category": "biolink:ChemicalSubstance"
+            }
+        },
+        "edges": {
+            "treats": {
+                "subject": "drug",
+                "predicate": "biolink:treats",
+                "object": "type-2 diabetes"
+            }
+        }
+    },
+    "knowledge_graph": {
+        "nodes": {
+            "MONDO:0005148": {"name": "type-2 diabetes"},
+            "CHEBI:6801": {
+                "name": "metformin",
+                "category": "biolink:ChemicalSubstance"
+            },
+            "CHEBI:5441": {
+                "name": "glyburide",
+                "category": "biolink:ChemicalSubstance"
+            }
+        },
+        "edges": {
+            "df87ff82": {
+                "subject": "CHEBI:6801",
+                "predicate": "biolink:treats",
+                "object": "MONDO:0005148"
+            },
+            "5133c100": {
+                "subject": "CHEBI:5441",
+                "predicate": "biolink:treats",
+                "object": "MONDO:0005148"
+            }
+        }
+    },
+    "results": [
+        {
+            "node_bindings": {
+                "type-2 diabetes": [{"id": "MONDO:0005148"}],
+                "drug": [{"id": "CHEBI:6801"}]
+            },
+            "edge_bindings": {
+                "treats": [{"id": "df87ff82"}]
+            }
+        }
+    ],
+    "processing_actions": {
+        "id": "overlay_compute_ngd", 
+        "parameters": {
+            "virtual_relation_label": "ngd1", 
+            "subject_qnode_key": "drug", 
```

Comment:
TODO: discuss duplicate keys (eg. A QG with two nodes of category `drug`)

Reply:
@dkoslicki Eric responded to this issue [here.](https://github.com/NCATSTranslator/ReasonerAPI/issues/230) Essentially it is not required that we use the exact category and in this case `drug1` and `drug2` is fine to use.

Reply:
Ah, gotcha, thanks!